keyboards/plywrks/allaro/info.json

```
@@ -0,0 +1,115 @@
+{
+    "manufacturer": "Plywrks",
+    "keyboard_name": "Allaro",
+    "maintainer": "ramonimbao",
+    "development_board": "promicro",
+
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true
+    },
+    "matrix_pins": {
+        "cols": ["F0", "F7", "F6", "C7", "C6", "B6", "B5", "B4", "D7", "D6", "D4", "D5", "D3", "D2", "D1"],
+        "rows": ["F1", "B7", "B0", "D0", "F5"]
+    },
+    "url": "",
+    "usb": {
+        "vid": "0x706C",
+        "pid": "0x7903",
+        "device_version": "1.0.0"
+    },
+    "rgblight": {
+        "led_count": 16,
+        "pin": "E2",
+        "hue_steps": 10,
+        "saturation_steps": 17,
+        "brightness_steps": 17,
+        "animations": {
+            "all": true
```

Could you explicitly enable the animation modes? 
```suggestion
            "alternating": true,
            "breathing": true,
            "christmas": true,
            "knight": true,
            "rainbow_mood": true,
            "rainbow_swirl": true,
            "rgb_test": true,
            "snake": true,
            "static_gradient": true,
            "twinkle": true
```